src/wabt.cpp

```diff
@@ -342,7 +342,7 @@ ExecutionResult WabtEngine::execute(
       const interp::TypedValues&,
       interp::TypedValues& results
     ) {
-      results[0].set_i32(static_cast<uint64_t>(interface.eeiGetCodeSize()));
+      results[0].set_i32(static_cast<uint32_t>(interface.eeiGetCodeSize()));
```

Comment:
This one in particular has a signature of `uint32_t` so wouldn't need a typecast: `uint32_t eeiGetCodeSize();`

Can you check if there are any other redundant typecasts?